shared/js/inject/chrome.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Inject all the overwrites into the page.
```

Comment:
This is the core file for protecting our code as we inject it into the page scope for chrome. We create a global function and add a password to prevent a page tampering.